consul/consul-k8s/consul.sysconfig

```diff
@@ -0,0 +1,3 @@
+CONFIG_DIR=/etc/consul
```

Comment:
Same reason we use variables for anything else: so you can change it without invoking `daemon-reload`.